app/controllers/users_controller.rb

```diff
@@ -13,11 +13,12 @@ def new
   end
 
   def edit
+    @user = current_user
   end
 
   def create
-    @user = User.new(user_params)
-
+    @user = User.create!(user_params)
+    @user.save
```

Comment:
good save!

Reply:
I thought that `.save` was only needed when used with `.new` and that `.create` essentially does both in one pass.

Reply:
You very well may be right. I know back in Mod 2, I ran into a ton of instances where my created whatevers didn't save, so I just got into the habit of adding the .save.